webapp/src/Controller/Team/SubmissionController.php

```diff
@@ -248,4 +248,10 @@ public function downloadAction(int $submitId): Response
 
         return $this->submissionService->getSubmissionZipResponse($submission);
     }
+
+    #[Route(path: '/submissions/team/{teamId<\d+>}/problem/{problemId<\d+>}', name: 'team_submissions')]
+    public function listAction(int $probId): Response
+    {
+
```

Comment:
huh, why is this empty?

Reply:
Originally I had a separate action for teams, but decided against it. I forgot to remove the action though.